src/PresentationalComponents/SystemsTable/SystemsTableAssets.js

```diff
@@ -20,6 +20,9 @@ export const systemsTableColumns = (intl) => [
       </React.Fragment>
     ),
   },
+  {
+    key: 'groups',
+  },
```

Comment:
see my previous comment

```suggestion
  {
    key: 'groups',
    requiresDefault: true
  },
```

Reply:
Added it :)